code/modules/events/disease_outbreak.dm

```diff
@@ -10,15 +10,23 @@
 
 	var/virus_type
 
+	var/max_severity = 3
 
-/datum/round_event/disease_outbreak/announce()
-	priority_announce("Confirmed outbreak of level 7 viral biohazard aboard [station_name()]. All personnel must contain the outbreak.", "Biohazard Alert", 'sound/ai/outbreak7.ogg')
+
+/datum/round_event/disease_outbreak/announce() //from level 7 onwards
+	priority_announce("Confirmed outbreak of level [4 + max_severity] viral biohazard aboard [station_name()]. All personnel must contain the outbreak.", "Biohazard Alert", 'sound/ai/outbreak7.ogg')

```

Comment:
changing the number doesn't make sense, since the sound will be the same and making it higher implies LESS threat, since blob is 5.
accordingly, do not change the number.

Reply:
I mean, it'd be nice if we had levels other than 5 and 7

Reply:
What @Cyberboss said. What's the point if there's only two levels and everybody knows one means blob and the other means some kind of unknown virus? This is much better imo.

Reply:
Would anyone be opposed to making blob a really high number? Or even a letter instead, to drive home the point of it being out of the scale in threat

Reply:
why even have blob be a number other than tradition at this point

blob being a level 5 biohazard
-doesn't tell noobs that it's a blob
-tells literally everyone else that it's blob

just make it say something else and use the biohazard level thing for this instead

Reply:
@Qbopper "Confirmed outbreak of hazardous class B biological mass."

Reply:
If nobody disagrees i'm gonna tack this onto the PR

Reply:
We don't have the sounds for it though!

Reply:
So, like, if you want to remove the sounds from virus and blob outbreaks then fine, go with that, but we only have those two sounds.

Reply:
Damn it i completely forgot, time to revert that for now, since i have zero clue on how to make the ai voice

Reply:
Literally *nobody* knows how to make the AI voice. The person who did it originally forgot, or was drunk, or something I don't actually remember, so we're stuck with this situation.

Reply:
....you're joking, right?

It's like the simplest effects on audacity.

I'll play around with it when I have some free time and make a guide so we can pimp our announcements.

Reply:
Nobody has ever been able to quite replicate the way the original clips sounded, thats why most of the other codebases swapped them out for their own standardised effecrs